src/dtemplate.d

```diff
@@ -633,9 +633,10 @@ public:
         return false; // don't scan uninstantiated templates
     }
 
-    override const(char)* kind()
+    override const(char)* kind() const
     {
-        return (onemember && onemember.isAggregateDeclaration()) ? onemember.kind() : cast(char*)"template";
+        auto om = cast(Dsymbol)this.onemember;    // strip const for the moment
```

Comment:
If you're going to insist on having the cast is should probably be the un-qualify cast: `cast()onemember`.